### ISSUE TITLE
Trivial suggestions

### ISSUE THREAD

**Issue by sleepylee** [zinzem/survey#9]
- Since leveraging kotlinx synthetic could reduce the boilerplate codes (findViewById), it would be better to be consistent and use it everywhere. For example, at your `NavigationIndicatorAdapter`:

Instead of:
```
view.findViewById<ImageView>(R.id.indicator)?.visibility = if (selected) VISIBLE else GONE
view.findViewById<ImageView>(R.id.indicator_empty)?.visibility = if (selected) GONE else VISIBLE
```

How about:
```
import kotlinx.android.synthetic.main.list_item_indicator.view.*

view.indicator.visibility = if (selected) VISIBLE else GONE
view.indicator_empty.visibility = if (selected) GONE else VISIBLE
``` 

- Also, talking of layout component retrieval, I'd suggest making the names say about itself and save our time on checking for its identity. For example: when reading from the kotlin class, we won't know whether the `indicator` is an ImageView or something else, and it takes time to check out the layout file. Instead, something like `ivIndicator` could tell us from the first read, how do you think? :)

**Comment by zinzem** [comment]
You're right about using kotlinx in the viewholder. I would just change it tho this:
```
import kotlinx.android.synthetic.main.list_item_indicator.view.*

view.indicator?.visibility = if (selected) VISIBLE else GONE
view.indicator_empty?.visibility = if (selected) GONE else VISIBLE
```
As the viewholder accepts any view, which might not contain these ids.

I also agree on naming, it helps. I see that you prefer `ivIndicator` to `iv_indicator`. I honestly can't choose. One looks better in the xml, the other in the kotlin code.

**Comment by sleepylee** [comment]
for the naming format convention, let's go with your own idea :D it's something always changeable depending on the team we're working with, but definitely, it should be consistently implemented throughout the whole code base.
